includes/options/class-amp-options-manager.php

```diff
@@ -115,28 +117,28 @@ public static function get_options() {
 			 *
 			 * @since 1.4.0
 			 */
-			$options['auto_accept_sanitization'],
+			$options[ Option::AUTO_ACCEPT_SANITIZATION ],
 			/**
 			 * Remove Story related options.
 			 *
 			 * @since 1.5.0
 			 */
-			$options['story_templates_version'],
-			$options['story_export_base_url'],
-			$options['story_settings'],
-			$options['enable_amp_stories'], // This was added in 1.2-beta and later migrated into the `experiences` option.
+			$options[ Option::STORY_TEMPLATES_VERSION ],
+			$options[ Option::STORY_EXPORT_BASE_URL ],
+			$options[ Option::STORY_SETTINGS ],
+			$options[ Option::ENABLE_AMP_STORIES ], // This was added in 1.2-beta and later migrated into the `experiences` option.
```

Comment:
I suppose this comment can be moved or added to to the option's docblock.

Reply:
Fixed via https://github.com/ampproject/amp-wp/pull/4545/commits/623bcbbeb1ff564269fe73f21459ea3a19089954